dbt-adapters/src/dbt/include/global_project/macros/materializations/models/incremental/on_schema_change.sql

```diff
@@ -101,7 +101,7 @@
 
 {% macro process_schema_changes(on_schema_change, source_relation, target_relation) %}
 
-    {% if on_schema_change == 'ignore' %}
+    {% if on_schema_change in ['ignore', 'full_refresh'] %}
```

Comment:
I agree with this, but we should also update `fail_msg` below. The second bullet is no longer needed and the first bullet should be updated with this option.